src/models/backend/chains.rs

```diff
@@ -28,3 +29,19 @@ pub struct Theme {
     pub text_color: String,
     pub background_color: String,
 }
+
+#[derive(Deserialize, Debug, PartialEq, Clone)]
+#[serde(tag = "type")]
+#[serde(rename_all = "lowercase")]
+pub enum GasPrice {
+    #[serde(rename_all = "camelCase")]
+    Oracle {
+        url: String,
+        gas_parameter: Option<String>,
```

Comment:
Why is this optional? I was thinking about if there is a default used by the clients :thinking: or does `null` imply something else on config service side?

In the typescript definition it is not nullable: https://github.com/gnosis/safe-react/blob/dev/src/config/networks/network.d.ts#L74

Reply:
It's optional (safe-config side) https://github.com/gnosis/safe-config-service/issues/87#issuecomment-865994250. I can change that.

Reply:
Since this matches what the safe-config service returns I'd say to tackle this in a different PR

Reply:
Gas parameter is now required: 86fb7a5